### PR TITLE
OOP school library: unit tests

### PR DESCRIPTION
**Pull request by Bondok6**
## OOP school library: unit tests

To run the test

```
rspec __tests__
```

In this update @Bondok6  and I [@JEFFKAY16]  added unit tests to these classes:

  - `Book`
  - `Person`
  - `Student`
  - `Teacher`
  - `Classroom`
  - `Corrector`
  - `Rental`

### CODE REVIEW

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Bondok6** [comment]
# Hello @t-yanick,
Thank you for your time, but I think there is something wrong.  
You did not explain, What issues do I still need to work on? 
I can't see any comments under the review.

**Comment by andersonlebon** [comment]
Hello @Bondok6  sorry what happened, maybe the reviewer did not remember to post his review comment here, But let me help you review this.

File: __tests__/person_spec.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    it 'return true if age is true or parent permission is true' do
+      person = Person.new(24)
+      expect(person.can_use_services?).to be(true)
+    end
+
+    it 'return true if age is true or parent permission is true' do
+      person = Person.new(24, name: 'Jeff')
+      expect(person.correct_name).to eql('Jeff')
+    end
```
Comment (andersonlebon) [inline __tests__/person_spec.rb]: ## OPTIONAL

- [ ] Kindly provide a good description of the test case.

**Review by andersonlebon — APPROVED** [review]
## Hi @
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: __tests__/person_spec.rb (hunk 1 above)
Comment (Bondok6) [reply __tests__/person_spec.rb]: Hello @andersonlebon 👋

Thanks for the review.
I'll fix it 👌

**Pull request merged**